### ISSUE TITLE
incoming calls doesn't work

### ISSUE THREAD

**Issue by opsidjflksdf** [seblj/roslyn.nvim#111]
Thank you for your work on this plugin!

Calling (while having the cursor on a method):
`:lua vim.lsp.buf.incoming_calls()`
displays 
`method textDocument/prepareCallHierarchy is not supported by any of the servers registered for the current buffer`

Would this be something easily fixable?

**Comment by seblj** [comment]
That should probably be reported to roslyn itself. I am not maintaining the server, as I am only starting the server and adding a few off-spec things roslyn have.

It seems like it has some code related to that, so maybe it will work if you update the server version.

However, I have no idea🤷‍♂️ not something I can do here at least

**Comment by drzbida** [comment]
there's something fishy going on, they are fully implemented in the server and somehow they don't work for us

even worse, not even :lua vim.lsp.buf.declaration() works with the same error that textDefinition/declaration is not supported by any of the servers

the capabilities generated generated by default_capabilities are correct 

maybe a custom hack like the one below is required, but I didn't find exactly what

```lua
-- HACK: Doesn't show any diagnostics if we do not set this to true
roslyn_config.config.capabilities = vim.tbl_deep_extend("force", roslyn_config.config.capabilities, {
    textDocument = {
        diagnostic = {
            dynamicRegistration = true,
        },
    },
})

```
